tavern/ent/schema/task.go

```diff
@@ -15,13 +15,30 @@ type Task struct {
 // Fields of the Task.
 func (Task) Fields() []ent.Field {
 	return []ent.Field{
-		field.String("name").
```

Comment:
No task names? Are we only using the unique ID to reference tasks?

Reply:
Jobs have the name, and tasks have an edge to the job